### ISSUE TITLE
One question

### ISSUE THREAD

**Issue by xuhao0721** [icesat-2UT/PhoREAL#28]
Hello, may I ask if the MSL in the file after ATL03 is converted from PhoREAL to CSV format is EGM2008？Thanks！

**Comment by icesat-2UT** [comment]
Yes! EGM2008 is the geoid model provided on the ATL03 data product.


From: xuhao0721 ***@***.***>
Sent: Saturday, March 30, 2024 9:25 PM
To: icesat-2UT/PhoREAL ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [icesat-2UT/PhoREAL] One question (Issue #28)


Hello, may I ask if the MSL in the file after ATL03 is converted from PhoREAL to CSV format is EGM2008？Thanks！

-

**Comment by xuhao0721** [comment]
Thanks a lot
